services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -198,16 +199,20 @@ const getProjectWebsite = record => {
  * @returns {number}
```

Comment:
Please update this statement. In the xml one `@returns {BudgetItem} The formatted budget` is ok i think